template/src/screens/demoSlice.ts

```diff
@@ -1,34 +1,12 @@
 import { PayloadAction, createSlice } from '@reduxjs/toolkit'
-import { call, put, takeEvery } from 'typed-redux-saga'
-import { getLatestComic } from '@api/comics'
-import { mapComic } from '@api/mappers/comicMappers'
-import { Comic } from '@api/types/comic.types'
 import { RootState } from '@redux/store'
-import { Failure, NotRequested, Pending, RemoteData, Success } from '@utils/api'
-import { getErrorMessage } from '@utils/error'
-
-export function* fetchLatestComic() {
-  try {
-    const comic = yield* call(getLatestComic)
-    yield* put(getLatestComicAsyncSuccess(mapComic(comic)))
-  } catch (error) {
-    const errorMessage = getErrorMessage(error)
-    yield* put(getLatestComicAsyncFailure(errorMessage))
-  }
-}
-
-export function* watchGetLatestComicSaga() {
-  yield* takeEvery(getLatestComicAsync, fetchLatestComic)
-}
 
 interface DemoState {
   counter: number
-  comic: RemoteData<Comic, Error['message']>
```

Comment:
No need to store `RemoteData` values (errors, loading) in our local state, because it's stored internally by `useQuery` in it's state that represents server state. This way networking doesn't mix with local state (MST or redux).